core/shared/mem-alloc/ems/ems_kfc.c

```diff
@@ -236,6 +236,13 @@ gc_migrate(gc_handle_t handle, char *pool_buf_new, gc_size_t pool_buf_size)
         return GC_ERROR;
     }
 
+    if ((uintptr_t)base_addr_new > (uintptr_t)heap->base_addr) {
+        offset = (uintptr_t)base_addr_new - (uintptr_t)heap->base_addr;
```

Comment:
The offset is used in adjust_ptr, if we use absolute value here, the adjust_ptr seems will return a wrong address?